2001-3000/LIT2365Storyo.xml

```diff
@@ -52,6 +52,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract><p>The Story of Melchizedek is an apocryphal account of the life of <persName ref="PRS6988Melchize"/>, an originally Greek text (spuriously attributed to <persName ref="PRS2213Athanasi"/>  in a
```

Comment:
Wouldn't PRS2209Athanasi fit better?

Reply:
the authority files are inherited from EAe index
there are many Athanasiuses in the index, each got his own ID
2213 = Athanasius, lit. IV:970b
I have checked, the entry is for Tamara tesbeet 
![image](https://github.com/user-attachments/assets/4c6ea978-aca1-47bd-812e-907cae5a9233)

so the keyword is for the _fictitious_ Athanasius, but indeed since the tradition identifies him with the patriarch probably the ID may be removed